controller_report_xls/controllers/main.py

```diff
@@ -1,27 +1,4 @@
 # -*- coding: utf-8 -*-
-###########################################################################
```

Comment:
@moylop260  Mmm.. interesting. I have only one question: We should use AGPL-3 or LGPL-3 when creating odoo modules? I use LGL-3 and in the example yu mentioned the module template has AGPL-3